src/pages/Payment/components/PaymentInfo.scss

```
@@ -0,0 +1,50 @@
+@import '../../../styles/common';
+@import '../../../styles/variables';
+
+.payment-info {
+  margin-top: 30px;
+  margin-bottom: 70px;
+  padding-left: 30px;
+  .text-xl {
+    padding-bottom: 15px;
+  }
+  .pay-box {
+    width: 320px;
```

```suggestion
  .text-xl {
    padding-bottom: 15px;
  }
  
  .pay-box {
    width: 320px;
```
스타일끼리는 가독성을 위해서 한칸씩 띄워주세요!